components/banner.js

```diff
@@ -1,12 +1,12 @@
-export const BlogBanner = ({ banner, small = false }) => {
+export const Banner = ({ banner, small = false, xsmall = false }) => {
```

Comment:
what if we need large xlarge? I mean boolean for defining the size is not enough in this case. Maybe we can use an enum here. For instance
```javascript
export const BannerSize = {
   SMALL: 'small',
   XSMAL: 'xsmall'
}


export const Banner = ({banner, size}) => {
   const getBannerHeight = () => {
       switch(size) {
          case BannerSize.SMALL:
               return 16.5;
          case BannerSize.XSMALL:
               return 9;
          default:
               return 21.25;
       }
   }
   return <>
       same code
       .
       .
       .
       height: ${getBannerSize()}em;
       .
       .
       .
   </>
}
```

and when we are using Banner we can do
```javascript
import {Banner, BannerSize} from '../path/to/banner';

const XPage = () => {
    return <Banner size={BannerSize.SMALL} .... />
}
```
What do you think?